civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -9,4 +9,106 @@
       }
     }
   }
+
+  .form-group {
+    display: inline-block;
+    width: 100%;
+
+    > label {
+      color: $crm-wizard-font-color;
+      display: block;
+      font-size: 13px;
+      padding: 0;
+      width: 100%;
+    }
+
+    > div {
+      padding: 0;
+      width: 100%;
+
+      div.form-item {
+        background-color: $light;
+        margin: 0;
+      }
+    }
+  }
+
+  .modal-civihr-custom__section {
+    margin-left: auto;
+    margin-right: auto;
+    position: relative;
+    top: -150px;
+    width: 475px;
+  }
+
+  .modal-civihr-custom__footer {
+    padding-top: 20px;
+  }
+
+  .page-header {
+    display: none;
+  }
+
+  .webform-previous {
+    background: $crm-wizard-primary-color;
+    border: 0;
+    color: $light;
+    float: left;
+
+    &:focus {
+      background: $crm-wizard-primary-color;
+      color: $light;
+    }
+  }
+
+  .webform-next {
+    background: $crm-wizard-secondary-color;
+    border: 0;
+    color: $light;
+
+    &:focus {
+      background: $crm-wizard-secondary-color;
+      color: $light;
+    }
+  }
+
+  .webform-component-markup {
+    background: transparent !important;
+
+    .onboarding-fieldset-title {
+      color: $crm-wizard-font-color;
+      margin-top: -15px;
+      font-size: 36px;
+      font-weight: 500;
+      text-align: center;
+    }
+  }
+
+  .webform-container-inline.webform-datepicker  div.form-item.form-type-select {
+    display: none;
+  }
+
+  .webform-calendar {
+    padding: 7px 10px;
+
+    &+.input-group-addon {
+      background-color: #F3F3F6;
```

Comment:
where did this color come from and why is it not a variable ?

Reply:
Changed to a closer color i.e `$crm_gray-light-alt`, so that I dont have to create a new variable.